v-next/hardhat/src/internal/builtin-plugins/network-manager/edr/utils/convert-to-edr.ts

```diff
@@ -212,23 +215,35 @@ export function edrRpcDebugTraceToHardhat(
   });
 
   // REVM trace adds initial STOP that Hardhat doesn't expect
+  // ASK: is this still valid?
```

Comment:
This isn't resolved

Reply:
@Wodann is going to double-check this, but for now it will remain as it is. I'll change the ASK for a TODO for easier tracking.